beerus_core/src/config.rs

```diff
@@ -42,13 +45,15 @@ impl Config {
         let starknet_core_contract_address = std::env::var("STARKNET_CORE_CONTRACT_ADDRESS")
             .unwrap_or_else(|_| DEFAULT_STARKNET_CORE_CONTRACT_ADDRESS.to_string());
         let starknet_core_contract_address = Address::from_str(&starknet_core_contract_address)?;
+        let data_dir = PathBuf::from("/tmp");
```

Comment:
Is the desired result that this is set by the user? If so we need to check an environment variable is set here.

Or can default to "/tmp" if none is set.

Reply:
Ok! Will move to .env